### ISSUE TITLE
Accessibility issue: ambiguous accessible label for previous and next button 

### ISSUE THREAD

**Issue by scoutb-cogapp** [UniversalViewer/universalviewer#1087]
**UV version:** _universalviewer@4.0.25_

**I'm submitting a:** *bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below*

### Page area
previous and next button inside image controls

### Issue description
These buttons have the title "Previous" and "Next" which is not as descriptive as it should be. A non-sighted user may not be aware that they are in an image viewer. It has never really been announced when the user entered the image viewer by tabbing. They would have to come to this conclusion by observing other buttons (like the zoom in and out buttons) and also conclude that the proximity to the other buttons means that the previous and next buttons also apply to an image. A lot of assumptions.

### Steps to reproduce
1. open [this manifest](https://www.universalviewer.dev/#?xywh=583%2C1062%2C2492%2C1379&iiifManifestId=https%3A%2F%2Feap.bl.uk%2Farchive-file%2FEAP500-1-1%2Fmanifest&cv=)
2. use a screenreader to navigate to the previous and next buttons inside the image viewer and observe what is announced

### Expected behaviour
Clear accessible labels that leave no doubt to users of assistive technology what they do.

### Possible fix
change the title attribute to "Next button" and "Previous button"

### WCAG criterion
[4.1.2 Name, Role, Value (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)

### Related code

```html
<div class="paging btn prev" tabindex="0" title="Previous" style="position: relative; padding-left: 0px; padding-top: 0px; display: inline-block; top: 239px;"></div>
```

### Other information
**Note**: This thinking could reasonably be applied to all the buttons in image controls but since something like zoom is usually assumed to apply to visuals, I have not raised it. It would not be a bad idea to go through the button names and labels and assess if their accessible lables can be improved.

**Comment by LlGC-mag** [comment]
Should be similar to #1075 so happy to take a look at this one too
